### PR TITLE
implemented code review changes 

### PR DESCRIPTION
**Pull request by Nesze**

### CODE REVIEW

**Comment by Zapadlo** [comment]
+1

File: ft-kafka_proxy/manifests/init.pp [hunk 1]
```diff
@@ -37,10 +37,11 @@
...
   exec {
     'restart-kafka-rest-proxy':
-      command		=> "/usr/bin/kafka-rest-stop || true && nohup /usr/bin/kafka-rest-start $config_file &",
+      command		=> "/usr/bin/kafka-rest-stop || nohup /usr/bin/kafka-rest-start $config_file &",
       subscribe		=> [ Package[$kafka_rest], File[$config_file], File[$log_file], Class['jdk'] ],
-      logoutput		=> true,
       refreshonly	=> true
   }
 
+  Exec['enforce_jdk_used'] ~> Service['kafka-proxy']
```
Comment (jinal--shah) [inline ft-kafka_proxy/manifests/init.pp]: Yikes! Sorry - I was providing an example. You don't actually have a Service resource for kafka-proxy. You're using an Exec instead to control the stop | start of the service.

In this case, line 45 should be:

```Exec['enforce_jdk_used'] ~> Exec['restart-kafka-rest-proxy']``` i.e. you want to trigger the stop-start action if java alternatives is reset.

Comment (Nesze) [inline ft-kafka_proxy/manifests/init.pp]: Doh, sorry

**Comment by jinal--shah** [comment]
Looks good.

I see that you're keeping the _|| true &&_ bit in the restart-kafka-rest-proxy Exec's command. Is that really required? (I'm curious why it would be)

**Comment by Nesze** [comment]
I'm afraid it is. Otherwise, when this command is triggered the service gets stopped, the OR condition evaluates to True, and the second part is not executed, because of short-circuit.

**Comment by tosan88** [comment]
+1

**Pull request merged**